static/app/components/charts/components/xAxis.tsx

```diff
@@ -31,13 +31,14 @@ function XAxis({
   addSecondsToTimeFormat = false,
   ...props
 }: Props): XAXisComponentOption {
-  const axisLabelFormatter = (value: string, index: number) => {
+  const AxisLabelFormatter = (value: string, index: number) => {
```

Comment:
Components are not allowed lowecase names, as you cannot call hooks inside "functions"